qa-core/src/internal-api/tests/dataSources/mariaDB.spec.ts

```diff
@@ -0,0 +1,71 @@
+import TestConfiguration from "../../config/TestConfiguration"
+import * as fixtures from "../../fixtures"
+
+describe("Internal API - Data Sources: MariaDB", () => {
+  const config = new TestConfiguration()
+
+  beforeAll(async () => {
+    await config.beforeAll()
+  })
+
+  afterAll(async () => {
+    await config.afterAll()
+  })
+
+  it("Create an app with a data source - MariaDB", async () => {
+    // Create app
+    await config.createApp()
+
+    // Add data source
+    const [dataSourceResponse, dataSourceJson] =
+      await config.api.datasources.add(fixtures.datasources.mariaDB())
+
+    // Update data source
+    const newDataSourceInfo = {
+      ...dataSourceJson.datasource,
+      name: "MariaDB2",
+    }
+    const [updatedDataSourceResponse, updatedDataSourceJson] =
+      await config.api.datasources.update(newDataSourceInfo)
+
+    const dataSourceQuery = {
+      datasourceId: updatedDataSourceJson.datasource._id,
+      fields: {
+        sql: "SELECT * FROM employees;",
+      },
+      name: "Query 1",
+      parameters: {},
+      queryVerb: "read",
+      schema: {},
+      transformer: "return data",
+    }
+    // Query data source
+
+    const [queryResponse, queryJson] =
+      await config.api.datasources.previewQuery(dataSourceQuery)
```

Comment:
Can we do some assertion on the `queryJson`? Like an expected response from this query.